### ISSUE TITLE
2023/03/21 meeting

### ISSUE THREAD

**Issue by hoyeon1234** [JBNU-JBIG/Paper-Review-Study#9]
1. 발표할 논문 + 요약 + 발표자료
2. 기타 공유할 것들

**Comment by hoyeon1234** [comment]
발표논문 : [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167)
발표자료 : [gitblog](https://hoyeon1234.github.io/sin-hoyeon/posts/paper%20study/batchnormalization/batchnormalization.html)

Batch단위의 normalization을 활용한 학습난이도 하강

**Comment by hsh6449** [comment]
1. GPT 1 (2,3,4 까지 커버하기엔 내용이 워낙 방대해서 나누기로 결정)
발표 논문 : Improving Language Understanding by Generative Pre-Training - [link](https://www.cs.ubc.ca/~amuham01/LING530/papers/radford2018improving.pdf)
발표 자료 : [notion](https://noble-stock-bf1.notion.site/Improving-Language-Understanding-by-Generative-Pre-Training-88ad748944514eef9c4c3a4479cffece)

2. stanford alpaca 디스코드 참조

**Comment by baekingeol** [comment]
발표 논문 : [BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/pdf/1810.04805.pdf)
발표 자료 : [notion](https://bony-chill-b63.notion.site/BERT-a4d9646e73d94972ae7ce68732fd5ff3)

**Comment by jwk1121** [comment]
1. FCN(Fully Convolution Network)
발표 자료: https://github.com/jwk1121/Study_PaperReview/tree/main/FCN
